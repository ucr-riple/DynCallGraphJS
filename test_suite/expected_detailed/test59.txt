{
    'global (test59.js:1:1:10:5)': [
        'anon (test59.js:1:2:10:2)'
    ],
    'anon (test59.js:7:3:9:5)': [
        'anon (test59.js:3:10:5:6)'
    ],
    'anon (test59.js:4:14:4:19)': [
        'anon (test59.js:7:19:9:4)'
    ]
}