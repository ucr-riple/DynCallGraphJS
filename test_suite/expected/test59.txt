{ '(test59.js:1:1:10:5)':
   [ '(test59.js:1:2:10:2)' ],
  '(test59.js:7:3:9:5)':
   [ '(test59.js:3:10:5:6)' ],
  '(test59.js:4:14:4:19)':
   [ '(test59.js:7:19:9:4)' ] }